content/metadata.yaml

```diff
@@ -155,7 +202,16 @@ authors:
     orcid: 0000-0003-4734-4508
     email: jaclyn.taroni@ccdatalab.org
     contributions:
-      - TODO
+      - Conceptualization
+      - Methodology
+      - Software
```

Comment:
Not that I remember!
```suggestion
```

Reply:
I was trying to give you some credit! I'm sure there's at least one contribution from you, but I'll remove it.